### ISSUE TITLE
Add channel density specifications in export_mitral.py

### ISSUE THREAD

**Issue by pgleeson** [OpenSourceBrain/MiglioreEtAl14_OlfactoryBulb3D#3]
As opposed to using chan densities etc. exported from Neuron directly, manually create the chan density elements in [export_mitral.py](https://github.com/OpenSourceBrain/MiglioreEtAl14_OlfactoryBulb3D/blob/master/NeuroML2/export_mitral.py). Will lead to neater cells once #2 is complete.

Info on biophys in https://github.com/OpenSourceBrain/MiglioreEtAl14_OlfactoryBulb3D/blob/master/NEURON/mkmitral.py etc.

See using libNeuroML to specify channel densities etc. here: https://github.com/OpenSourceBrain/AllenInstituteNeuroML/blob/master/CellTypesDatabase/models/ParseAll.py#L128

See example of cell in nml2 here: https://github.com/OpenSourceBrain/ACnet2/blob/master/neuroConstruct/generatedNeuroML2/pyr_4_sym.cell.nml

**Comment by JustasB** [comment]
@pgleeson What is the state on the existing converted channels? Are they usable as is, or are there issues? 

I'm referring to these under NeuromML2 folder:
kamt.channel.nml
kdrmt.channel.nml
nax__sh15.channel.nml
nax.channel.nml
pas.channel.nml

**Comment by JustasB** [comment]
@pgleeson I'm having some issues with paths. Works fine when I run  export_mitral.py from NEURON folder, but not NeuroML2 folder. I'm assuming it should work from NeuroML2 folder.

**Comment by pgleeson** [comment]
You can consider the channels stable. There are some tests for these in here: https://github.com/OpenSourceBrain/OlfactoryTest/tree/master/NeuroML2

There is some hopping between folders in the python scripts, due to the location of the hoc files. Best to compile mods in the NEURON folder, go into ../NeuroML2, then do: 'nrnivmodl ../NEURON' also...

**Comment by JustasB** [comment]
@pgleeson Is there some example code out there for getting channel mechanisms from hoc files in python? It's not as straightforward as I first expected.

**Comment by pgleeson** [comment]
I wouldn't do it by parsing and extracting the values programatically. Just create the channelDensities manually in libNeuroml and cut and paste/convert the correct values in export_mitral.py...

**Comment by JustasB** [comment]
Ah, that simplifies it a lot. Ok will do.

**Comment by JustasB** [comment]
Channel densities are now part of mitral cell NML files.

Implemented here: https://github.com/OpenSourceBrain/MiglioreEtAl14_OlfactoryBulb3D/pull/9

Note: Since we're manually building the biophys part of the cell NML, I manually created the NML portion of the biophys element in separate XML files that get substituted in after segments are generated. See the NeuroML2/MitralCell...xml files and the last few lines of main() in export_mitral.py. Let me know if this is fine. Seems that it makes it easier to read and modify later than if the same xml elements are constructed in python.
